### PR TITLE
feat(netapp): Handle master filers list

### PR DESCRIPTION
**Pull request by mathis-marcotte**
for https://jirab.statcan.ca/browse/BTIS-481

### CODE REVIEW

**Comment by mathis-marcotte** [comment]
Here is an example of the code logic working https://goplay.tools/snippet/i6Qm-qLjIkp

File: cmd/ontap-cvo.go [hunk 1]
```diff
@@ -487,22 +484,8 @@ func performHttpPost(username string, password string, url string, requestBody [
...
-	svmInformation := svmInfo{
-		svmName: svmName,
-		svmUUID: svmUUID,
-		svmUrl:  svmURL,
-	}
-	return svmInformation
-}
-
 // Format JSON data helper function
+// TODO: is this needed or should this be removed?
```
Comment (Jose-Matsuda) [inline cmd/ontap-cvo.go:488]: I am now thinking that it can be removed, given that we have like all our functionality and didnt need it

**Review by Jose-Matsuda — APPROVED** [review]
lgtm

File: cmd/ontap-cvo.go (hunk 1 above)
Comment (mathis-marcotte) [reply cmd/ontap-cvo.go:488]: Ill remove it now

**Pull request merged**